sqlglot/dialects/databricks.py

```diff
@@ -51,6 +68,10 @@ class Generator(Spark.Generator):
             exp.ToChar: lambda self, e: self.function_fallback_sql(e),
         }
 
+        def computedcolumnconstraint_sql(self, expression: exp.ComputedColumnConstraint) -> str:
```

Comment:
can you link the documentation for computed column constraint? why is generated always as a computed column constraint?

Reply:
https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql?view=sql-server-ver16 I think the clause of interest is `computed_column_definition` in the spec.